arch/arm64/boot/dts/qcom/msm8953-xiaomi-vince.dts

```diff
@@ -128,6 +128,12 @@
 	compatible = "xiaomi,vince-panel";
 };
 
+&pmi8950_haptics {
+	status = "okay";
+
+	qcom,wave-play-rage-us = <5263>;
```

Comment:
I don't know how relevant this PR still is but since it is not closed:

This line contains a bit too much `rage` instead of `rate`. It should probably be `qcom,wave-play-rate-us`.